src/nni_manager/core/nnimanager.ts

```diff
@@ -325,6 +325,10 @@ class NNIManager implements Manager {
         // FIXME: unit test
     }
 
+    public async getTrialLog(trialJobId: string, logType: LogType): Promise<string> {
+        return this.trainingService.getTrialLog(trialJobId, logType);
```

Comment:
it might be better to catch `MethodNotImplementedError` here, then return log message like `XXX training service does not support retrieving log`.

Reply:
Currently only local mode will display buttons, I think maybe it'll be better to throw this error to remind developer when developing this feature on other training services.

Reply:
could you explain how did you fix this one?

Reply:
In the latest commit, I revert this to the original version (i.e., not handle the error here) since the front end will check the training service now. The error here could also be an file operation error, so I let the restHander to catch the error throwed here.